_clean.R

```diff
@@ -4,6 +4,14 @@ snd <- snd %>%
   mutate(alt_st_name = str_trim(gsub("\\s+", " ", alt_st_name))) %>%
   mutate(full_stname = str_trim(gsub("\\s+", " ", full_stname)))
 
+# parse out lat and lng from "the_geom" in building footprints (bin centroids) and cast to doubles
+bincentroids <- bincentroids %>%
+  mutate(the_geom = gsub("POINT \\((.*)\\)", "\\1", "POINT (-73.87136418648842 40.657210892463695)")) %>%
```

Comment:
there are of course parsers for this format (WKT) but regex is way easier since we're just dealing with points

Reply:
word. but also this is clearly wrong b/c im replacing every single `the_geom` with my parsed test string :O will fix

Reply:
fixed